UnityProject/Assets/Scripts/Core/Database/PersistenceServer.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Core.Database
+{
+	public static class PersistenceServer
+	{
+		public static string Host = "dev-api.unitystation.org"; // TODO: expose to config
```

Comment:
@MaxIsJoe 
wasn't this exposed as a config already?

Reply:
it was, but something reverted it I believe; or this is an entirely different class that manually writes its own host thing